src/include/cephfs/libcephfs.h

```diff
@@ -39,9 +39,9 @@ using CephContext = ceph::common::CephContext;
 extern "C" {
 #endif
 
-#define LIBCEPHFS_VER_MAJOR 10
+#define LIBCEPHFS_VER_MAJOR 11
```

Comment:
If the changes only add new interfaces, shouldn't it be better to increase just the minor ?

Reply:
This what we discussed with the team and decided to bump major since it was not changed for the last 10 years. We want to make it now a baseline for the future changes.

Reply:
hmm I also thought we were only bumping the major version on incompatible changes. @vshankar did you have a different opinion on this?

Reply:
The argument being that you can check the major version to know very clearly if your code (compiled at some arbitrary point) will still run against the library, or if it's become incompatible.

Reply:
@vshankar WDYS?

Reply:
The rules are pretty loose here. I suggested @salieri11 to bump the major version actually. But now I recall, the interfaces are new (the old implementation changed the ll_preadv_writev interface. So, I'm fine with bumping the minor version.